src/microbe_stage/FloatingChunk.cs

```diff
@@ -110,14 +110,13 @@ public override void _Ready()
         }
 
         AddChild(GraphicsScene.Instance());
+        AddToGroup(Constants.FLUID_EFFECT_GROUP);
```

Comment:
Chunks are spawned by a spawn helper in Spawners.cs the group applying should be put there:
https://github.com/Revolutionary-Games/Thrive/blob/17265383d1af4a94997fddb1673f19c5cbc28ee0/src/microbe_stage/Spawners.cs#L79
similarly to how the microbe spawner does it:
https://github.com/Revolutionary-Games/Thrive/blob/17265383d1af4a94997fddb1673f19c5cbc28ee0/src/microbe_stage/Spawners.cs#L50